vgen/src/main/resources/io/gapi/vgen/py/main.snip

```diff
@@ -212,18 +217,10 @@
                 object through which to make calls.
               ssl_creds (:class:`grpc.beta.implementations.ClientCredentials`):
                 A `ClientCredentials` for use with an SSL-enabled channel.
-              retrying_override (dict[string, :class:`google.gax.RetryOptions`]): A
-                dictionary that overrides default retrying settings.
-                ``retrying_override`` maps method names (e.g., ``'list_foo'``) to
-                custom RetryOptions objects, or to None. A value of None indicates
-                that the method in question should not retry.
-              bundling_override (dict[string, :class:`google.gax.BundleOptions`]): A
-                dictionary that overrides default bundling settings.
-                ``bundling_override`` maps bundling method names (e.g.,
-                'publish_foo') to custom BundleOptions objects, or to None. It is
-                invalid to have a key for a method that is not bundling-enabled. A
-                value of None indicates that the method in question should not
-                bundle.
+              client_config (dict or :class:`google.gax.CallSettings`):
```

Comment:
If a dict is specified, preferably we should use it to update the default client config. That way you can pass in a partial specification.

CallSettings also includes attribute that are not intended to be overridden by the user, like PageDescriptor and BundleDescriptor, so I'm not sure it's the right object to use here.